jwt/test_jwt.txt

```diff
@@ -0,0 +1,3 @@
+token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJyb2xlIjoic2VydmljZV9yb2xlIiwiaWF0IjoxNjM2MzcxNDkzLCJleHAiOjE5NTE5NDc0OTN9.T365h6AcgnvibAWvN_rIiaFnOK5R1ZGqdEkPI45zsNs
+token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiYXVkIjoiYXRsYXNzaWFuIiwiaWF0IjoxNTE2MjM5MDIyLCJhbm90aGVyX2ZpZWxkIjoidmFsdWUifQ.ez5bJeF93WXSU-dCWlZVFlTl_inJCYVFFLOo1JBIsh0
```

Comment:
Seems this is being picked up as a `Atlassian JSON Web Token`. Not a problem, just an observation.

https://github.com/advanced-security/secret-scanning-custom-patterns/security/secret-scanning/502